packages/agreement-process/src/model/domain/validators.ts

```diff
@@ -381,8 +382,8 @@ const attributesSatisfied = <
 
 const matchingAttributes = (
   eServiceAttributes: EServiceAttribute[][],
-  consumerAttributes: string[]
-): string[] =>
```

Comment:
In this `validators.ts` file there are many places where we declare functions with `agreementId: string` params (and maybe other Ids?) - consider replacing them with the new type?